src/topp/SequenceCoverageCalculator.cpp

```diff
@@ -123,7 +129,7 @@ class TOPPSequenceCoverageCalculator :
     // calculations
     //-------------------------------------------------------------
 
-
+    map<String, CoverageInfo> prot2cov;
```

Comment:
I know you are always a fan of the regular map, but I think for strings and potentially thousands of them, I think unordered really would be more efficient.